src/app/components/line-chart/line-chart.component.html

```diff
@@ -0,0 +1,28 @@
+<div>
+    
+    <!-- <button class="btn btn-outline-dark rounded-0" (click)="showRoute1()">Route 1</button>
```

Comment:
Let's endeavor to not include any commented out code in the codebase. It just keeps everything clean.